src/posix/termios.rs

```diff
@@ -163,6 +163,26 @@ pub(crate) fn set_parity(termios: &mut Termios, parity: Parity) {
             termios.c_iflag |= libc::INPCK;
             termios.c_iflag &= !libc::IGNPAR;
         }
+        Parity::Mark => {
+            termios.c_cflag |= libc::PARODD;
+            termios.c_cflag |= libc::PARENB;
+            termios.c_iflag |= libc::INPCK;
+            termios.c_iflag &= !libc::IGNPAR;
+            #[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
Wouldn't this just silently switch to normal odd even parity checking other backends?

Reply:
Yes. I'm not sure what is the best way to handle that. Making the whole enum variant conditionally available has some problems as I've noted above. Another option could be to panic if it is not supported, which is arguably better than failing silently.